src/components/landing/Testimonials/TestimonialsCloud.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+
+import Avatar from './Avatar';
+
+import ronAvatar from '../../../../content/assets/home/testimonial/ron-yotpo/avatar.webp';
+import ronAvatarJpeg from '../../../../content/assets/home/testimonial/ron-yotpo/avatar.jpeg';
+
+import markAvatar from '../../../../content/assets/home/testimonial/mark-hopper/avatar.webp';
+import markAvatarJpeg from '../../../../content/assets/home/testimonial/mark-hopper/avatar.jpeg';
+
+import jonAvatar from '../../../../content/assets/home/testimonial/jon-ncsa/avatar.webp';
+import jonAvatarJpeg from '../../../../content/assets/home/testimonial/jon-ncsa/avatar.jpeg';
+
+const TESTIMONIALS = [{
+  title: 'Incredible',
+  text: (
+    <>
+      <p className="prose mb-2">The Roadie team have been incredible to work with. They’re obviously Backstage experts, and their platform provides us with a ton of flexibility and integrations.</p>
+      <p className="prose">We&apos;ve been able to start using Backstage must faster and we don’t have to worry about the maintenance.</p>
```

Comment:
Using `&apos;` here but not above or later
```suggestion
      <p className="prose">We've been able to start using Backstage must faster and we don’t have to worry about the maintenance.</p>
```

Reply:
Linter complains if I do that.

Reply:
That seems weird how it complains on one thing but not the second one

Reply:
It makes a bit more sense now hopefully.